Student/delivery/http/student.go

```diff
@@ -142,3 +142,117 @@ func (h *StudentHandler) Delete(c *gin.Context) {
 
 	c.JSON(http.StatusOK, httputils.NewResponse("student deleted"))
 }
+
+func (h *StudentHandler) AddClasses(c *gin.Context) {
+	id := c.Param("id")
```

Comment:
I noticed line 147 till 166 are duplicated in AddClasses, RemoveClasses, CompleteClass, can you extract this id check into an "isLoggedIDAuthorized" method?